proto/internal.go

```diff
@@ -82,3 +83,23 @@ func (m *RaftMessageRequest) GetUser() string {
 	// TODO(marc): we should use security.NodeUser here, but we need to break cycles first.
 	return "node"
 }
+
+// NewRaftMessageRequest returns a new RaftMessageRequest.
+func NewRaftMessageRequest(groupID RangeID, msg raftpb.Message) (*RaftMessageRequest, error) {
```

Comment:
can you put a TODO somewhere to change RaftMessageRequest.Msg to raftpb.Message in the proto and move the proto into multiraft?